Stripe/PublicHeaders/STPSourceEnums.h

```diff
@@ -162,9 +162,35 @@ typedef NS_ENUM(NSInteger, STPSourceType) {
      A WeChat Pay source. @see https://stripe.com/docs/sources/wechat-pay
      */
     STPSourceTypeWeChatPay,
+    
+    /**
+     A Klarna source. @see https://stripe.com/docs/sources/klarna
+     */
+    STPSourceTypeKlarna,
 
     /**
      An unknown type of source.
      */
     STPSourceTypeUnknown,
 };
+
+/**
+ Custom payment methods for Klarna
+ @see https://stripe.com/docs/sources/klarna#create-source
+ */
+typedef NS_OPTIONS(NSUInteger, STPKlarnaPaymentMethods) {
+    /**
+     Don't specify any custom payment methods.
+     */
+    STPKlarnaPaymentMethodsNone = 0,
+
+    /**
+     Offer payments over 4 installments. (a.k.a. Pay Later in 4)
+     */
+    STPKlarnaPaymentMethodsPayIn4 = 1 << 0,
+
+    /**
+     Offer payments over an arbitrary number of installments. (a.k.a. Slice It)
+     */
+    STPKlarnaPaymentMethodsInstallments = 1 << 1,
```

Comment:
Why are these options instead of an enum? Does it make sense to combine `PayIn4` and `Installments`?

Reply:
The API specifies that you can provide both, which will offer the user a choice of PayIn4 and Installments. It doesn't appear to do this in practice. :/

Reply:
gotcha